### DISCUSSION TITLE
error:  offset outside the bounds of dataview

### DISCUSSION THREAD

**Discussion by RoyGalaxy** [google/model-viewer#4251] [Q&A] [answered]
### Description
I am using model-viewer to diplay 3d models(.glb format) of food items for a restraunt webiste. Model-viewer works fine on the localhost but when i tried the same on an actual server it gives me an error. I tried using some other models(.glb format) but in that case it gave me errors as well.

Here is a clip from my console
![Screenshot_20230425_234408_Samsung Internet](https://user-images.githubusercontent.com/75137220/234367084-9b679714-7d46-4b87-acbc-387f62bae3f6.jpg)

I also tried searching for this error but couldn't find any solution. Kindly respond to my problem.

#### Live Demo
https://erin-bee-kit.cyclic.app/model.html

### Version
- model-viewer: v3.0.1

### Browser Affected
I am not sure about other browsers but I got this error on my chrome(android) as well as samsung internet(android)

### OS

- [x] Android 13

**Comment by elalish** [comment]
Your GLB is invalid: https://github.khronos.org/glTF-Validator/ Please file a bug on whatever program exported it.

**Reply by RoyGalaxy** [reply]
well as per your suggestion I tried changing the GLB file(size: 7MB ), i tried saveral other files which i downloaded from sketchfab, now I am resulting in a fetch error

![Screenshot_20230426_232507_Samsung Internet](https://user-images.githubusercontent.com/75137220/234662832-a33fa56d-ee06-434f-beb3-600f6264f5d7.jpg)

can it be becuase of my hosting provider ? should I try any other provider ?

becuase the new GLB files and the old GLB file all works fine on localhost. Also the old GLB file worked fine on the server as well, till the last month. From this month I started getting these type of error.

I also tried the gltf validator from the link of above message, and it is saying that my gltf files are all valid

**Reply by elalish** [reply, marked as answer]
I don't know - try the files on our editor first to make sure that's all working. If so, then it's some kind of server issue.

**Reply by RoyGalaxy** [reply]
Okay, thanks for help
